resources/assets/js/components/HRPortal/__snapshots__/JobIndexHr.test.tsx.snap

```diff
@@ -216,10 +215,10 @@ exports[`Job Index Page should render JobIndexHr correctly 1`] = `
             }
           }
           key="4"
+          reviewRequested="2019-MAY-02"
```

Comment:
Is this snapshot how the fixtures are rendered? It doesn't look like they're in newest -> oldest by review here.

Reply:
Ahh yes, the unclaimed jobs list gets sorted in `JobIndexHrPage.tsx` which is a wrapper before passing data to `JobIndexHr.tsx`. I can change it to use JobIndexHrPage component instead.

Reply:
Actually it's easier if I just sort it within the after it gets passed into `JobIndexHr.tsx`